common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -148,8 +148,8 @@ trait PerformanceSwitches {
 
   val DiscussionSwitch = Switch(
     SwitchGroup.Performance,
-    "discussion",
-    "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
+    "discussion-visible-web",
+    "If this switch is on, comments are displayed on articles on web. This switch only removes the UI from the articles on web and does not affect requests to DAPI.",
```

Comment:
I guess it would reduce POSTs as there is not the form to submit comments. Just it will have a limited impact on GETs?